boards/shared/Test/Src/App_TestWorld.c

```diff
@@ -0,0 +1,15 @@
+#include "App_TestWorld.h"
+
+#include <stdlib.h>
+
+struct TestWorld {};
```

Comment:
I believe our compiler flags aren't happy about empty structs (But I assume you'd find out about that eventually anyway)

Reply:
The test seems to compile fine, and this isn't ever included in an `arm` executable, so I think it should be Ok?

Reply:
Oh yeah you're right, I assumed it's part of the arm target because it's `.c`